frontend/src/views/CarViews/CarLifecycleView.js

```diff
@@ -0,0 +1,184 @@
+import React, { Component, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { List } from '@material-ui/core';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListItemAvatar from '@material-ui/core/ListItemAvatar';
+import Avatar from '@material-ui/core/Avatar';
+import DriveEtaIcon from '@material-ui/icons/DriveEta';
+import PersonIcon from '@material-ui/icons/Person';
+import LocalGasStationIcon from '@material-ui/icons/LocalGasStation';
+import EmojiObjectsIcon from '@material-ui/icons/EmojiObjects';
+import InvertColorsIcon from '@material-ui/icons/InvertColors';
+import DiscFullIcon from '@material-ui/icons/DiscFull';
+import BuildIcon from '@material-ui/icons/Build';
+import NotificationsActiveIcon from '@material-ui/icons/NotificationsActive';
+import ReceiptIcon from '@material-ui/icons/Receipt';
+import ChromeReaderModeIcon from '@material-ui/icons/ChromeReaderMode';
+
+class CarLifecycleView extends Component {
+    
+    render() {
+        return (
```

Comment:
Could we have this dynamic. Otherwise we can't really use it.

set up a mock object, and load elements from it . i.e.:
```
render() {
...
let list = {
//structure
}

//and below in return, ex: 
return(

    <div>
    {
        list.map( (item) => {
              <ELEMENT>{item} </ELEMENT>
        })
    }
   </div>
)
```

Not everything needs to be put inside **map** function, only the repeating parts, which in this case are many